src/codeu/chat/client/simplegui/ChatGuiFX.java

```diff
@@ -21,19 +20,18 @@
     private final static Logger.Log LOG = Logger.newLog(ChatGuiFX.class);
 
 
-    private static final double WIDTH = 1000;
-    private static final double HEIGHT = 500;
+    private static final double WINDOW_WIDTH = 1000;
+    private static final double WINDOW_HEIGHT = 500;
 
     private Stage thestage;                         // Holds the scene that user is currently viewing
     private Scene signInScene, mainScene;           // Scenes to hold all the elements for each page
-    private BorderPane signInPane;                  // Root panes for both pages (panes provide structure to scene elements)
-    private FlowPane signInLabelPane;               // Panes to hold labels (so that they can be aligned in the root pane)
-    private GridPane inputPane;                     // Pane to position username/password input fields and labels, as well as sign-in button
-    private Label signInLabel;                    // Labels for the main page and sign in page
-    private Button signInButton;                    // Buttons to switch between main page and sign in page
-    private TextField userInput, passInput;         // Text fields for entering username and password
+    private Button signInButton;
```

Comment:
Do all of these variables need to be class-level variables? I think you might be able to move some of them to be local variables. Looks like you started this process with some of the other variables already.